fe/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -379,6 +384,14 @@ public boolean getForwardToMaster() {
         return forwardToMaster;
     }
 
+    public void setDefaultRowsetType(String defaultRowsetType) {
```

Comment:
`setDefaultRowsetType` and `getDefaultRowsetType` are both unused, remove them

Reply:
done